.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Test suite
+
+on:
+  pull_request:
+    branches: [ devel ]
+  push:
+    branches:
+      - master
+
+jobs:
+  quality-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Code quality checks
+        run: |
+          docker build --build-arg PYTHON_VERSION=3.6 --build-arg DJANGO_VERSION=1.11 -t native -f Dockerfile-tmpl .
+          docker run --rm native sh -c 'pre-commit run --files $(git ls-files transifex* tests*)'
+
+  python2-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Test py2.7-dj1.11
+        run: |
+          docker build --build-arg PYTHON_VERSION=2.7 --build-arg DJANGO_VERSION=1.11 -t native -f Dockerfile-tmpl .
+          docker run -e CODECOV_TOKEN=$CODECOV_TOKEN --rm native sh -c 'pytest --cov --cov-report=term-missing && codecov'
+        env:
+          CODECOV_TOKEN: ${{secrets.codecov_token}}
```

Comment:
Any reason this is lowercase but `{{secrets.PYPI_API_TOKEN}}` below is uppercase?